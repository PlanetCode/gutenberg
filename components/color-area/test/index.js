/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import ColorArea from '../';

describe( 'ColorArea', () => {
	it( 'matches the snapshot', () => {
		const wrapper = shallow( <ColorArea ariaLabel="sample label" colorValue="#fff" /> );

		expect( wrapper ).toMatchSnapshot();
	} );
} );
