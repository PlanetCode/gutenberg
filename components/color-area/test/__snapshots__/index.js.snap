// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorArea matches the snapshot 1`] = `
<span
  aria-label="sample label"
  className="component-color-area"
  style={
    Object {
      "background": "#fff",
    }
  }
/>
`;
